librenms-service.py

```diff
@@ -35,6 +35,7 @@
     try:
         service = LibreNMS.Service()
     except Exception as e:
+        raise
```

Comment:
By design?

Reply:
Bah!  I had that in for troubleshooting (to get stacktraces).  We might want to update this section to actually raise if debug is set?

Reply:
Just remove it for now.  Make a different PR for that if you want to make that change.